.github/workflows/pylint.yml

```diff
@@ -14,42 +14,28 @@ jobs:
       uses: actions/setup-python@v3
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install Java
+    - name: Install Dependencies
       run: |
-        sudo apt install default-jdk
-    - name: Install ANTLR
-      run: |
-        wget https://www.antlr.org/download/antlr-4.13.1-complete.jar
-        echo 'export CLASSPATH=".:$GITHUB_WORKSPACE/antlr-4.13.1-complete.jar:$CLASSPATH"' >> $GITHUB_ENV
-    - name: Install pip dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install antlr4-python3-runtime pylint
-    - name: Build ANTLR files
-      run: |
-        cd build && make antlr-java
-        cd ..
+        source .github/common-setup.sh
     - name: Pylint style analysis
       run: |
         echo "Running pylint on source code..."
         pylint $(git ls-files 'src/*.py') \
```

Comment:
I think ideally we should aim to not disable any of these, but consider that more of a stretch goal than a requirement.

Reply:
I agree. Long term we should aim to disable as many Pylint warning suppressions as we can.